catalog/app/components/Assistant/UI/Chat/DevTools.tsx

```diff
@@ -0,0 +1,63 @@
+import * as Eff from 'effect'
+import * as React from 'react'
+import * as M from '@material-ui/core'
+
+import JsonDisplay from 'components/JsonDisplay'
+
+import * as Model from '../../Model'
+
+const useStyles = M.makeStyles((t) => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    maxHeight: '100%',
+  },
+  heading: {
+    ...t.typography.h5,
+    borderBottom: `1px solid ${t.palette.divider}`,
+    lineHeight: '64px',
```

Comment:
> I think, you wanted to match the height of the footer? 

you mean header? no, that wasn't my intention, i just wanted to make it look nice relative to the close/menu icon button, so that it has even spacing below and above, and it does.

it actually does match the header height as well, tho the border/divider adds one pixel -- i think that's ok, since the header has a solid(ish) background and is separated from the rest of the page with the shadow, which is, too, _below_ the body of the header, so ima keep this as is